docs/open-source-project/release-notes/tizen-5-0-m2.md

```diff
@@ -0,0 +1,468 @@
+  - Removed wifi dependency and added wifi-manager dependency.
+- Maps-plugin-here 
+  - Updated.
+  - Fixed memory leak and potential defects.
+  - Updated user consent.
+- Sensor 
+  - Fixed memory leak and potential defects.
+- Email 
+  - Fixed memory leak and potential defects.
+- Contact
+  - Disable unsupported charset.
+  - Fixed potential defect.
+
+
+## Web framework
+
+### New and changed features
+
+- Add feature of [App control with URI scheme]
+  - Support feature which is similar with android chrome intent.
+  - Tizen native app implicit/explicit launching by using [app-control:// ]
+  - Supporting Fallback url.
+- Implement alternative TBM back-end for web page rendering.
+  - By setting chromium-efl’s layout result can be rendered to memory surface.
+  - Using this path chromium=efl webview can be merged with given UI framework (NUI,..).
+
+
+## Lightweight Web Solution
+
+### New and changed features
+
+- WebView support for Bixby Capsule Viewer on TV has been added.
+- WebView support for DALi (NUI) has been added.
+
+
+## Tizen .NET
+
+### New and changed features
+
+- .NET Core (Runtime)
+  - Coreclr version 2.1.1 is upgraded to 2.1.4.
+  - Corefx version 2.1.1 is upgraded to 2.1.4.
+- Xamarin.Forms
+  - Xamarin.Forms 3.2.0 which is latest stable version is supported.
+- Tizen.CircularUI
+  - Wearable UI extension (Tizen.Wearable.CircularUI) version 1.0.0-pre2 is supported.
+- Tizen.TV.UIControls
+  - TV UI extension (Tizen.TV.UIControls) version 1.0.0-beta is now available.
+- TizenFX
+  - Added new MediaController APIs for playlist, event and capabilities.
+  - Added new MediaController APIs for sending command and its callback and deprecate old one.
+  - Added new AudioManager APIs for USB Audio output device.
+  - Deprecated the device state enums of AudioManager.
+  - Added new AudioManager APIs for device running changed event and deprecate old one.
+  - Added new AudioIO APIs and enum for sample type and change maximum sample rate.
+  - Added new MediaPlayer APIs to set/get the video ROI area.
+  - Added new Camera APIs for new preview format (RGB).
+  - Added new Multimedia APIs for the rotation of MediaTool.
+  - Added new Multimedia APIs to set/get playback position in nanoseconds.
+  - Added new Multimedia APIs to set zoom level with the field of view.
+  - Added new Multimedia APIs to set/get replaygain.
+  - Added new Multimedia APIs to set/get adaptive variants and property for buffering time.
+  - Added new Nlp APIs of new Namespace and Class.
+  - Added new InputMethod APIs for floating keyboard and to hide/launch IME.
+  - Added new WiFi APIs for forgetting AP and, BssidScan, RawSsid, CountryCode, WPS connection cancellation.
+  - Added new Applications APIs for rpc-port to support RPC (Remote Procedure Call).
+  - Added new NUI APIs for KeyboardRepeatInfo and TextPredition.
+  - Added the change of NUI APIs for ChildAdded, ChildRemoved, PropertySet events & properties of TableView.CellPosition.

```

Comment:
Can you elaborate further?
Please use "and" instead of "&".

Reply:
It's also good.
Please make it so.

Reply:
will accept